pkg/operator/ceph/cluster/mon/config.go

```diff
@@ -160,10 +161,46 @@ func CreateOrLoadClusterInfo(clusterdContext *clusterd.Context, context context.
 		clusterInfo.CephCred.Username = string(secret.Data["userID"])
 		clusterInfo.CephCred.Secret = string(secret.Data["userKey"])
 	}
+	if err := ValidateCephCSIConnectionSecrets(clusterInfo.Context, clusterdContext, namespace); err != nil {
+		return nil, maxMonID, monMapping, err
+	}
 
 	return clusterInfo, maxMonID, monMapping, nil
 }
 
+// ValidateCephCSIConnectionSecrets returns the secret value of the client health checker key
+func ValidateCephCSIConnectionSecrets(ctx context.Context, clusterdContext *clusterd.Context, namespace string) error {
```

Comment:
Validating the csi secrets is unrelated to the context change. Was this an intentional change? If so, would you mind separating this for a follow up PR?